trace-reader/Readme.md

```diff
@@ -0,0 +1,36 @@
+# trace-reader
+
+## Introduction
+This program reads picscope .trace files, a binary file developed by E.M Schooneveld.
+
+## Command Line Interface
+trace-reader [OPTIONS] --broker <BROKER>
+
+### Options:
+```
```

Comment:
This is available via `--help` so does not need to be repeated here (where it can only get out of date).